samples/nrf9160/http_update/src/update.c

```diff
@@ -13,20 +13,32 @@
 #include <modem/bsdlib.h>
 #include <modem/modem_key_mgmt.h>
 #include <net/fota_download.h>
-#include <dfu/mcuboot.h>
+#include <net/socket.h>
+#include <nrf_socket.h>
+#include <stdio.h>
 
-#define LED_PORT	DT_GPIO_LABEL(DT_ALIAS(led0), gpios)
+#define LED_PORT DT_GPIO_LABEL(DT_ALIAS(led0), gpios)
 #define TLS_SEC_TAG 42
 
-static const struct	device *gpiob;
-static struct		gpio_callback gpio_cb;
-static struct k_work	fota_work;
+static const struct device *gpiob;
+static struct gpio_callback gpio_cb;
+static struct k_work fota_work;
 
+void (*finished_cb)(void);
+const char *(*host_get)(void);
+const char *(*file_get)(void);
```

Comment:
Minor: Why not use "get" as the prefix here to make it more statement-like?

Reply:
In NCS we typically place the verb at the end. So I think the way it is now has the highest probability of being approved in the next round of PRs ;)

Reply:
Noted, I'll keep that in mind for my own code.